spec/rails_helper.rb

```diff
@@ -1,9 +1,6 @@
-if ENV["RCOV"]
-  require "simplecov"
-  require "simplecov-rcov"
```

Comment:
Can we also drop this from the Gemfile?

Reply:
Oops, missed that this time round - updated 👍